gluoncv/model_zoo/icnet.py

```diff
@@ -0,0 +1,381 @@
+# pylint: disable=unused-argument,abstract-method,missing-docstring,arguments-differ
+"""Image Cascade Network (ICNet)"""
+from __future__ import division
+from mxnet.gluon import nn
+from mxnet.context import cpu
+from mxnet.gluon.nn import HybridBlock
+from gluoncv.model_zoo.segbase import SegBaseModel
+from gluoncv.model_zoo.pspnet import _PSPHead
+from gluoncv.loss import SoftmaxCrossEntropyLoss
+
+
+__all__ = ['ICNet', 'get_icnet', 'get_icnet_resnet50_citys', 'ICNetLoss']
+
+
+class ICNetLoss(SoftmaxCrossEntropyLoss):
```

Comment:
I think this is a loss function, which should be put into this file, https://github.com/dmlc/gluon-cv/blob/master/gluoncv/loss.py